docs/design/README.md

```diff
@@ -2145,6 +2152,10 @@ default package.
 
 ### Imports
 
+> **Note:** This is provisional, designs for making the package name optional
+> have not been through the proposal process yet. See
+> [#2323](https://github.com/carbon-language/carbon-lang/issues/2323).
+
```

Comment:
Is there a way to phrase this so it is clear that, while it hasn't been through the proposal process, it was decided in #1136 ?

Reply:
Adjusted to reference #2001 for tracking.